test/translate-adapter.test.ts

```diff
@@ -2,8 +2,8 @@ import { expect } from "chai";
 import * as dircompare from "dir-compare";
 import { copy } from "fs-extra";
 import path from "path";
-import rimraf from "rimraf";
-import { promisify } from "util";
+import { rimraf } from "rimraf";
+//import { promisify } from "util"; // promisify not needed with rimraf ^4.0.0, rimraf has promise
```

Comment:
YPu can simply remove these comments here and below because irrelevant for the future ...

Reply:
DONE